apps/tup-cms/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "root": "apps/tup-cms/",
+  "sourceRoot": "apps/tup-cms/src/",
+  "projectType": "application",
+  "targets": {
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "npx nx build tup-ui",
+          "docker-compose -f docker-compose.dev.yml build"
```

Comment:
should this be a production build command? also i think this may be causing the pipeline to hang since it is doing `nx build affected`

Reply:
Good call, I need to go in and mess with the pipeline config to make sure we skip the docker build.